themes/qgrid/layout/partials/nav.ejs

```diff
@@ -1,89 +1,27 @@
 <nav>
-    <h3>Getting started</h3>
-
     <ul>
-        <li>
-            <a href="#">Introduction</a>
-        </li>
-        <li>
-            <a href="#">How to install</a>
-        </li>
-        <li>
-            <a href="#">Concepts and architecture</a>
-        </li>
-        <li>
-            <a href="#">Grid model and service</a>
-        </li>
-    </ul>
+        <% var groupOrder = ['Getting started', 'API', 'Features'];
+        var groupped = site.pages.reduce(function (groups, p) {
+            var key = p.group;
+            (groups[key] = groups[key] || []).push(p);
+            return groups;
+        }, {});
 
-    <h3>Features</h3>
-    <ul>
-        <li>
-            <a href="#">Sorting</a>
-        </li>
-        <li>
-            <a href="#">Filtering</a>
-        </li>
-        <li>
-            <a href="#">Groupping and aggregates</a>
-        </li>
-        <li>
-            <a href="#">Pivoting</a>
-        </li>
-        <li>
-            <a href="#">Scrolling</a>
-        </li>
-        <li>
-            <a href="#">Pipes</a>
-        </li>
-        <li>
-            <a href="#">Grid appearance</a>
-        </li>
-        <li>
-            <a href="#">Templates</a>
-        </li>
-        <li>
-            <a href="#">Data editing</a>
-        </li>
-        <li>
-            <a href="#">Data selection and usability</a>
-        </li>
-        <li>
-            <a href="#">Grid navigation</a>
-        </li>
-        <li>
-            <a href="#">State persistance</a>
-        </li>
-        <li>
-            <a href="#">Action model</a>
-        </li>
-        <li>
-            <a href="#">Plugins</a>
-        </li>
-        <li>
-            <a href="#">Mobile device support</a>
-        </li>
-        <li>
-            <a href="#">Localization</a>
-        </li>
-        <li>
-            <a href="#">Context menu</a>
-        </li>
-    </ul>
+        for (var g = 0; g < groupOrder.length; g++) {
+            var key = groupOrder[g];
 
-    <h3>API</h3>
-    <ul>
-        <li>
-            <a href="#">One term</a>
-        </li>
-        <li>
-            <a href="#">Another point</a>
-        </li>
-        <li>
-            <a href="#">Third unit</a>
-        </li>
-        <li>
-            <a href="#">And so on</a>
+        if (groupped.hasOwnProperty(key)){ %>
+        <li><h3><%- key %></h3></li>
+        <% var sorted = groupped[key].sort(function (a, b) {
```

Comment:
sorted -> pages
add variable `var page = groupped[key] `

Reply:
do sort requires to return boolean or `-1, 0, 1` ? (a.order - b.order) ?

Reply:
boolean